### PR TITLE
updating fbcs to 0.0.2

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
Can confirm all SHAs are correct here, we just need to discover why our `redis` and `rekor-search-ui-rhel9` are not built
/cc @osmman @petrpinkas @ppecka @lance 
Relates to: https://redhat-internal.slack.com/archives/C05M2GGKU7Q/p1707450833732149?thread_ts=1707449931.383969&cid=C05M2GGKU7Q

### CODE REVIEW

File: fbc/v4.11/catalog/rhtas-operator/catalog.json [hunk 1]
```diff
@@ -243,11 +243,11 @@
         },
         {
             "name": "",
-            "image": "registry.redhat.io/rhtas-tech-preview/rekor-search-ui-rhel9@sha256:325f1e84936c31e02bddb2bc4fff07c3a55c2e556deba72e99f4ec99aa589cca"
```
Comment (Gregory-Pereira) [inline fbc/v4.11/catalog/rhtas-operator/catalog.json:246]: This registry coordinate does not exist but it should: registry.redhat.io/rhtas-tech-preview/rekor-search-ui-rhel9@sha256:325f1e84936c31e02bddb2bc4fff07c3a55c2e556deba72e99f4ec99aa589cca

File: fbc/v4.11/catalog/rhtas-operator/catalog.json [hunk 2]
```diff
@@ -243,11 +243,11 @@
...
+            "image": "quay.io/redhat-user-workloads/rhtas-tenant/rekor-1-0-gamma/rekor-search-ui-1-0-gamma@sha256:325f1e84936c31e02bddb2bc4fff07c3a55c2e556deba72e99f4ec99aa589cca" // previously: registry.redhat.io/rhtas-tech-preview/rekor-search-ui-rhel9@sha256:325f1e84936c31e02bddb2bc4fff07c3a55c2e556deba72e99f4ec99aa589cca but this does not exist atm
         },
         {
             "name": "",
-            "image": "registry.redhat.io/rhtas-tech-preview/redis-trillian-rhel9@sha256:914089e0e8407420ce07f44dad49da75d34c88eac6314ea8d6e45ff0745e4b42"
```
Comment (Gregory-Pereira) [inline fbc/v4.11/catalog/rhtas-operator/catalog.json:250]: This registry coordinate does not exist but it should: registry.redhat.io/rhtas-tech-preview/redis-trillian-rhel9@sha256:914089e0e8407420ce07f44dad49da75d34c88eac6314ea8d6e45ff0745e4b42

**Review by Gregory-Pereira — COMMENTED** [review]
These two comments extend to every catalog overlay

File: fbc/v4.11/catalog/rhtas-operator/catalog.json (hunk 2 above)
Comment (tommyd450) [reply fbc/v4.11/catalog/rhtas-operator/catalog.json:250]: Found it buried within our comet entries, [here](https://catalog.redhat.com/software/containers/detail/65b155240bfd129442f53b4d?tag=0.0.2-1707432078&architecture=amd64&image=65c55a503e1c38056b20b692&container-tabs=overview)

File: fbc/v4.11/catalog/rhtas-operator/catalog.json (hunk 1 above)
Comment (tommyd450) [reply fbc/v4.11/catalog/rhtas-operator/catalog.json:246]: This one was published briefly after you posted this as you mentioned in slack but just for posterity and clarity for anyone getting to this pr in the future, [Link to search ui cataloge entry](https://catalog.redhat.com/software/containers/rhtas-tech-preview/rekor-search-ui-rhel9/65b15173431374bd574c4829?architecture=amd64&image=65c50a1b77c9221b875f8ba9)

Comment (tommyd450) [reply fbc/v4.11/catalog/rhtas-operator/catalog.json:246]: Gotcha update all catalogs with images for rekor-search and redis.

**Comment by tommyd450** [comment]
Adding the rest of the 0.0.2 images now

**Comment by tommyd450** [comment]
The images with their respective shas exist on registry now. 
Going to see if there is a successful build on konflux and then merge.

**Pull request merged**